src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -212,9 +229,46 @@ def ramp_fit_data(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d,
         nframes = ramp_data.nframes
         readnoise_2d *= gain_2d / np.sqrt(2. * nframes)
 
+        # Suppress one group ramps, if desired.
+        if ramp_data.suppress_one_group_ramps:
+            suppress_one_group_ramps(ramp_data)
```

Comment:
Within this call to suppress_one_group_ramps(ramp_data), intdq marks the groups of pixels having only a single group. It's not obvious to me how that information gets passed back into ramp_fit_data( ) - via ramp_data  ?

Reply:
The `intdq` array is essentially a pointer to a subarray of `ramp_data.groupdq`.  Any value change to `intdq` is a value change in `ramp_data.groupdq`.